src/spatial_query/pipeline.rs

```diff
@@ -474,7 +474,7 @@ impl SpatialQueryPipeline {
 
     /// Casts a [shape](spatial_query#shapecasting) with a given rotation and computes computes all [hits](ShapeHitData)
     /// in the order of distance, calling the given `callback` for each hit. The shapecast stops when
-    /// `callback` returns false or all hits have been found.
+    /// `callback` returns true or all hits have been found.
```

Comment:
I'm pretty sure the old docs are correct here. The cast continues until `false` is returned by the callback, like Rapier's [`intersections_with_shape`](https://docs.rs/bevy_rapier3d/0.28.0/bevy_rapier3d/plugin/struct.RapierContext.html#method.intersections_with_shape).